test/parallel/test-http-chunked-304.js

```diff
@@ -24,20 +24,39 @@ function test(statusCode, next) {
   });
 
   server.listen(common.PORT, function() {
-    var conn = net.createConnection(common.PORT, function() {
-      conn.write('GET / HTTP/1.1\r\n\r\n');
+    var 
```

Comment:
Linting fails due to trailing space on this line. Can you fix that and use `make jslint` to confirm no other oddities slipped in?